src/common/api/common/CommonMailUtils.ts

```diff
@@ -0,0 +1,41 @@
+// @bundleInto:common
```

Comment:
you probably don't need the annotation anymore?

Reply:
No, I don't. Removed